frontend/src/components/LandingPage/LandingPage.tsx

```diff
@@ -97,7 +98,9 @@ const LandingPage: React.FC<Verification> = ({verify}: Verification) => {
         <div className="description">
             <p>Verify to start moving</p>
```

Comment:
As a whole, the change looks good to me, but could we also put the "Verify to start moving" text inside the conditional, since it only makes sense with the button? Also, remove the `<h1>` tag.

```js
{!isVerified &&
  <>
    <p>Verify to start moving</p>
    <button onClick={ () => googleLogin() }>Sign in with Google</button>
  </>
}
```

In the future, we may want different text or UI displayed here when verified so that the landing page doesn't look too bare, but we can make this part of the UX improvement tasks.